hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -1436,16 +1451,17 @@ class Emit[C](
 
             val codeB = emit(body, env = bodyenv)
             Code(xElt := elt,
-              tmpAcc := codeB.map(v => accType.copyFromPValue(mb, region, PCode(body.pType, codeB.v))),
+              tmpAcc := codeB.map(v => accType.copyFromPValue(mb,
+                                                              region.code, PCode(body.pType, codeB.v))),
```

Comment:
is this generated automatically from your IDE? I think our typical style settings would break params onto the next line + 1 indent, not at the same level of indentation.

Reply:
Oops, yeah. I tried to fix them up after the refactoring, but missed this one.